nncf/dynamic_graph/graph.py

```diff
@@ -168,10 +168,30 @@ def call_order(self):
         return self.input_agnostic.call_order
 
 
+class BaseAttributes:
+    def __init__(self, weight_requires_grad: bool):
+        self.weight_requires_grad = weight_requires_grad
+
+
+class ConvolutionAttributes(BaseAttributes):
+    def __init__(self,
+                 weight_requires_grad: bool,
+                 in_channels: int,
+                 out_channels: int,
+                 stride: Tuple[int, int],
+                 groups: int):
+        super().__init__(weight_requires_grad)
+        self.in_channels = in_channels
+        self.out_channels = out_channels
+        self.stride = stride
+        self.groups = groups
+
+
 class NNCFNode:
-    def __init__(self, node_id: int, op_exec_context: OperationExecutionContext):
+    def __init__(self, node_id: int, op_exec_context: OperationExecutionContext, attributes: BaseAttributes = None):
         self.node_id = node_id
         self.op_exec_context = op_exec_context
+        self.attributes = attributes
```

Comment:
Now this description is too generic.
->`module_attributes`

Reply:
Done